lib/matplotlib/lines.py

```diff
@@ -581,7 +581,7 @@ def set_markevery(self, every):
         -----
         Setting the markevery property will only show markers at actual data
         points.  When using float arguments to set the markevery property
-        on irregularly spaced data, the markers will likely not appear evenly
+        on irregularly spaced data, the markers will may not appear evenly
         spaced because the actual data points do not coincide with the
```

Comment:
```suggestion
        on irregularly spaced data, the markers may not appear evenly
        spaced because the actual data points do not coincide with the
```